osbs/build/user_params.py

```diff
@@ -280,6 +284,15 @@ def set_params(self,
         self.signing_intent.value = signing_intent
         self.compose_ids.value = compose_ids or []
 
+        try:
+            self.orchestrator_deadline.value = string.atoi(orchestrator_deadline)
+        except (ValueError, TypeError):
+            self.orchestrator_deadline.value = 4
```

Comment:
It would be nice to have these fallback values in the constants module

Reply:
Fixed.